htdocs/install/mysql/data/llx_c_product_nature.sql

```diff
@@ -0,0 +1,29 @@
+-- Copyright (C) 2020 florian HENRY florian.henry@scopen.fr
+--
+-- This program is free software; you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation; either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with this program. If not, see <https://www.gnu.org/licenses/>.
+--
+--
+
+--
+-- Do not place a comment at the end of the line, this file is parsed when
+-- from the install and all '--' are removed.
+--
+-- Ne pas placer de commentaire en fin de ligne, ce fichier est parsé lors
+-- de l'install et tous les sigles '--' sont supprimés.
+--
+
+INSERT INTO llx_c_product_nature (code, label, active) VALUES (-1, '', 1);
```

Comment:
Some old dictionnary has this but it is a bad practive to add a value empty into list of possible value.
This must instead addressed when we suggest the combo list by adding a paramater "$showempty".